src/queries/useChunkedCQLFetch.test.js

```diff
@@ -194,4 +199,37 @@ describe('Given useChunkedCQLFetch', () => {
       expect(result.current.itemQueries?.length).toEqual(2);
     });
   });
+
+  describe('allows work in provided tenant', () => {
+    const providedTenantId = 'providedTenantId';
+
+    it('sets up 1 fetch using alternate ID name', async () => {
```

Comment:
More copy paste problems: you're not changing the ID name here, you're changing the tenantId. How about "sets up 1 fetch using alternate tenant ID"?